routes/user.js

```diff
@@ -1,10 +1,11 @@
 const express = require("express");
 const router = express.Router();
 
-const { createUser, deleteUser, fetchUserByID } = require("../controllers/user");
+const { createUser, deleteUser, fetchUserByID, updateUserByID } = require("../controllers/user");
 
 router.post("/users", createUser);
 router.get("/users/:id", fetchUserByID);
 router.delete("/users/:id", deleteUser);
+router.put("/users/:id", updateUserByID);
```

Comment:
probably we should use the verify token middleware for these routes

Reply:
Done, added middleware to certain endpoints from user resource (GET, PUT, DELETE)